src/main/java/spark/RouteWrapper.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2015 - Per Wendel
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package spark;
+
+import spark.Filter;
+import spark.FilterImpl;
+import spark.Request;
+import spark.Response;
+import spark.Route;
+import spark.RouteImpl;
+
+/**
+ * Utility methods for wrapping routes.
+ */
+class RouteWrapper {
```

Comment:
I agree. The name sucks, I'll change it.

I don't remember every little change that has been made. I believe it was to make the transition to the Spark 2.0 API smoother. This code could probably be made redundant by doing changes in SparkInstance.